presto-docs/src/main/sphinx/spill/overview.rst

```diff
@@ -0,0 +1,67 @@
+Spill to disk
+=============
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Overview
+--------
+
+In the case of memory intensive operations, Presto features offloading
+intermediate operator results to disk. The goal of this mechanism is to
+enable executing queries requiring amounts of memory exceeding per query
+or per node limits.
+
+The mechanism is similar to OS level page swapping. However, it is
+reimplemented on the application level to address specific needs of Presto.
+
+Properties related to spilling are described in properties chapter
+:ref:`spill properties<tuning-spill>`.
+
+Revocable memory management
+---------------------------
+
+By default, Presto manages memory needed by drivers manually to guarantee
```

Comment:
Do not use `drivers`. It is internal concept. Users will not understand that probably.